docs/applications/configuration-management/Using-Laravel-Forge-with-Linode.md

```diff
@@ -0,0 +1,226 @@
+
+**Server Size** The size of the server that you'd like to create. Here we have many different options so pick whatever fits the needs for your application. This is the server option in Linode.
+
+**Region** Wherever you want your server to be located. Ideally you'd want it to be close to where you believe your main users will be.
+
+**PHP Version** This will be the php version that will be installed on your Linode server.
+
+**Database Name** If your application needs a database, then you can name it here. By default it will be named `forge`.
+
+2. Once you have the configuration you like, go ahead and click Create Server. 
+
+   [![Linode API](/docs/assets/configuration-management/server_credential.png)](/docs/assets/configuration-management/server_credential.png)
+
+When you click on `create server`, you will get a pop up showing your sudo password and database password. There are the details of the database that has been created for you.
+
+Now, your newly created server will be listed in the Active server below. You should now see that Forge is going through the paces of creating a server, configuring it, and getting it ready based on the settings we just input.
+
+Once the new server is ready to go, you will see it's green and good to go(By default, it takes some seconds).
+
+
+[![Linode API](/docs/assets/configuration-management/active_servers.png)](/docs/assets/configuration-management/active_servers.png)
+
+When you visit your IP Address, you will see that our server has been set up for us.
+
+[![Linode API](/docs/assets/configuration-management/server_set_up_with_php_7_1.png)](/docs/assets/configuration-management/server_set_up_with_php_7_1.png)
+
+When the server is active, you will also receive mail from Linode with details of things that has been created.
+
+[![Linode API](/docs/assets/configuration-management/mail_from_linode_showing_details_created.png)](/docs/assets/configuration-management/mail_from_linode_showing_details_created.png)
+
+You will also receive a mail that a public key was added to your gitlab account.
+
+Now, lets confirm that the server is created on Linode also, head over to linode under the linode tab, you should see a node that has been created.
+
+
+[![Linode API](/docs/assets/configuration-management/linode_created.png)](/docs/assets/configuration-management/linode_created.png)
+
+
+## Pointing your domain name
+
+If you intend to visit the site you're setting up via a real domain (versus just testing it by visiting the IP address), you have to add the domain because DNS record changes can take a while to update. 
+
+1. Now head back to Laravel Forge account then click on the active server. 
+
+2. Go to your domain name provider, add an ANAME record for the domain or subdomain, and point it to the public IP address of your server. Your public IP address is as seen in the screenshot below, mine is: `41.33.92.166` That's it!
```

Comment:
Do not use personal IP address. Substitute with example IP address per style guide: https://github.com/linode/docs/blob/master/docs/linode-writers-formatting-guide.md#example-ip-addresses